src/main/java/dev/morling/onebrc/CalculateAverage_ricardopieper.java

```diff
@@ -0,0 +1,437 @@
+package dev.morling.onebrc;
```

Comment:
Please run mvn license:format to add the missing header.

Reply:
Done!